code/ui/generalhud/nameplate/Nameplate.cs

```diff
@@ -90,7 +90,7 @@ public override void Tick()
             }
 
             TraceResult trace = Trace.Ray(player.EyePos, player.EyePos + player.EyeRot.Forward * MAX_DRAW_DISTANCE)
-                .Ignore(player)
+                .Ignore(player.CurrentPlayer)
```

Comment:
I think we need 2 Ignores: Ignore the player and the player.CurrentPlayer if player.IsSpectating